.github/workflows/update-pot-files.yml

```diff
@@ -0,0 +1,38 @@
+
+name: Update source *.pot files from source repo
+on:
+  schedule:
+    - cron: '45 0 * * *' # at 0:45 UTC
+
+  # allow to run the workflow manually
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Update *.pot files
```

Comment:
Ditto the job name, and for the ID (which is now `build`), maybe use something more fitting, `build` is probably a copy-paste remnant. `update-pots` or `update-source-translations` maybe?